src/Microsoft.CodeAnalysis.Analyzers/Core/MetaAnalyzers/Fixers/SourceGeneratorAttributeAnalyzerFix.cs

```diff
@@ -90,9 +90,7 @@ private static async Task<Document> FixDocumentAsync(Document document, SyntaxNo
                 {
                     RoslynDebug.Assert(languageNames[i] == LanguageNames.CSharp || languageNames[i] == LanguageNames.VisualBasic);
 
-                    var language = languageNames[i] == LanguageNames.CSharp
-                        ? nameof(LanguageNames.CSharp)
-                        : nameof(LanguageNames.VisualBasic);
+                    var language = languageNames[i];
 
```

Comment:
Sorry I think I was incorrect here. The constant value is "C#", it doesn't match the field constant name ("CSharp").

Reply:
You know, I trusted you Youssef. 😑

I'll get this tomorrow as well when it's not bedtime 😉

Reply:
Sorry again.

Actually, even the UICulture suggestion is failing CI (I don't quite understand why). But the failure is for the good this time. If it's wrong to use UICulture for `string.Format`, then this should get updated and the analyzer should be enabled in roslyn:

https://github.com/dotnet/roslyn/blob/b88948f10a1085980df7080f0b765b1a8c614436/src/Compilers/CSharp/Portable/Compiler/DocumentationCommentCompiler.cs#L289

Reply:
I was just jesting, it's easy to make that mistake. I often forget that "C#" is what the string is. 

As for UICulture, I'll look into that. I would expect it to be correct, but it's possible not formatting for culture at all is more correct in these cases.

Reply:
> src\Microsoft.CodeAnalysis.Analyzers\Core\MetaAnalyzers\Fixers\SourceGeneratorAttributeAnalyzerFix.cs(38,21): error CA1305: (NETCORE_ENGINEERING_TELEMETRY=Build) The behavior of 'string.Format(string, object)' could vary based on the current user's locale settings. Replace this call in 'SourceGeneratorAttributeAnalyzerFix.RegisterCodeFixesAsync(CodeFixContext)' with a call to 'string.Format(IFormatProvider, string, params object[])'

Here it is... This seems to be disabled during build in VS though. @sharwell is the correct thing to do is pass `CurrentCulture` here?